src/assistant_library_demo.py

```diff
@@ -60,7 +61,8 @@ def process_event(event):
 
 def main():
     credentials = aiy.assistant.auth_helpers.get_assistant_credentials()
-    with Assistant(credentials) as assistant:
+    device_id, model_id = aiy.assistant.device_helpers.get_ids(credentials)
```

Comment:
not that when using the `google-assistant-library` it generates its own ID (and you have no way to pass another one).